src/app/client/projects/desktop/src/app/modules/offline/components/offline-help-videos/offline-help-videos.component.spec.ts

```diff
@@ -22,4 +59,27 @@ xdescribe('OfflineHelpVideosComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should initialize slide data', fakeAsync(() => {
+    const resourceService = TestBed.get(ResourceService);
+    resourceService.instance = resourceServiceStub.instance;
+    resourceService.frmelmnts = resourceServiceStub.frmelmnts;
+    component.ngOnInit();
+    tick(500);
+    expect(component.slideData).toBeDefined();
+  }));
+  it('should changeVideoAttributes value', fakeAsync(() => {
+    const data = component.slideData[0];
+    component.changeVideoAttributes(data);
```

Comment:
this should be triggered by querying all video card element and clicking on one of them. And we need assertion to check if that element data is played not the other video